projects/demo/src/modules/experimental/swipe-action/examples/1/index.less

```diff
@@ -0,0 +1,8 @@
+[tuiSurface] {
+    margin: 1.5rem 1rem;
```

Comment:
We need to show an example where we add margin to the surface and negative margin to swipe action to compensate, so that somebody can, for example, control distance between items with flex gap.

Reply:
I added custom gap to example with reduced size of items + custom actions-padding to second example